rest/__init__.py

```diff
@@ -1,45 +1,44 @@
+from __future__ import absolute_import
 import flask
 import unicodecsv as csv
 
-from exceptions import ValueError
-
 from flask import Response
 from functools import wraps
-from StringIO import StringIO
+from io import BytesIO
 from werkzeug.wrappers import BaseResponse
 
-from schema import Schema
-
-from model import Model
-
-from fields import Bool
-from fields import DateTime
-from fields import Dict
-from fields import Dollars
-from fields import Email
-from fields import Float
-from fields import Int
-from fields import List
-from fields import NoneInt
-from fields import NoneString
-from fields import ReadOnly
-from fields import String
-from fields import StringBool
-from fields import TruthyOnlyList
-from fields import URL
-from fields import WriteOnce
-from fields import WriteOnly
-
-from validators import email
-from validators import length
-from validators import multiple_choice
-from validators import nonempty
-from validators import number_range
-from validators import regex
-from validators import required
-from validators import url
-
-from encoding import encoder
+from .schema import Schema
+
+from .model import Model
+
+from .fields import Bool
+from .fields import DateTime
+from .fields import Dict
+from .fields import Dollars
+from .fields import Email
+from .fields import Float
+from .fields import Int
+from .fields import List
+from .fields import NoneInt
+from .fields import NoneString
+from .fields import ReadOnly
+from .fields import String
+from .fields import StringBool
+from .fields import TruthyOnlyList
+from .fields import URL
+from .fields import WriteOnce
+from .fields import WriteOnly
+
+from .validators import email
+from .validators import length
```

Comment:
is this syntax backwards compat with py2?

Reply:
yeah this was done by python-modernize, i believe it added the from __future__ import absolute_import
to handle this https://www.python.org/dev/peps/pep-0328/